pkg/gui/controllers/helpers/refs_helper.go

```diff
@@ -587,3 +581,19 @@ func (self *RefsHelper) ParseRemoteBranchName(fullBranchName string) (string, st
 func IsSwitchBranchUncommittedChangesError(err error) bool {
 	return strings.Contains(err.Error(), "Please commit your changes or stash them before you switch branch")
 }
+
+func (self *RefsHelper) GetSuggestedBranchName(suggestedBranchName string) (string, error) {
```

Comment:
The function doesn't need to be exported, so it can be lowercase. See fixups e0fd8ede6cde and c6523e6545e3.